Week01/Problem05/qxu_05.py

```diff
@@ -0,0 +1,18 @@
+# define gcd function
+def gcd(x, y):
```

Comment:
Nice work! I appreciate the extra effort it took to learn (or at least recall) the Euclidean algorithm, and code it up.  You could have achieved the same result, however, by importing the `math` module, like so:
```
import math
```
and then using the greatest common divisor function from this module with `math.gcd()`. That way, you are less likely to encounter bugs, since you don't have to go to the trouble of writing your own GCD function.